From 22eaa4ec2419b1a1e55bc269e8106f32f6a5f752 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Mon, 11 Oct 2021 13:55:31 +0200
Subject: [PATCH] kernel: Enable AHCI in default kernel

This enables option to boot from another type of external storage.
---
 target/linux/mvebu/config-4.14 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/target/linux/mvebu/config-4.14 b/target/linux/mvebu/config-4.14
index 294edd58ec..edcf60eeaa 100644
--- a/target/linux/mvebu/config-4.14
+++ b/target/linux/mvebu/config-4.14
@@ -1,4 +1,5 @@
 CONFIG_AHCI_MVEBU=y
+CONFIG_SATA_AHCI=y
 CONFIG_ALIGNMENT_TRAP=y
 CONFIG_ARCH_CLOCKSOURCE_DATA=y
 CONFIG_ARCH_HAS_DEBUG_VIRTUAL=y
-- 
2.33.0

